impl/src/main/java/org/ehcache/config/CacheConfigurationBuilder.java

```diff
@@ -136,4 +142,59 @@ private CacheConfigurationBuilder(CacheConfigurationBuilder<? super K, ? super V
   public boolean hasDefaultExpiry() {
     return expiry == null;
   }
+
+  public CacheConfigurationBuilder<K, V> withEvictionVeto(EvictionVeto<K, V> evictionVeto) {
```

Comment:
If that one is added to align on the `with*` naming pattern, then `evictionVeto` one should be removed